src/components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -85,6 +94,26 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
 
   const [state, dispatch] = useReducer(resourceFormReducer, initialState);
 
+  const { data: facilitiesResponse } = useQuery<
+    PaginatedResponse<FacilityData>
+  >({
```

Comment:
types are automatically inferred from the route's definition as provided here: https://github.com/ohcnetwork/care_fe/blob/develop/src/types/facility/facilityApi.ts#L10

```suggestion
  const { data: facilitiesResponse } = useQuery({
```